ADOPTERS.md

```diff
@@ -0,0 +1,37 @@
+# Adopters
+
+**📢 If you're using JSON Schema in your organization, please add your company name to this list.**
+**🙏 It really helps the project to gain momentum and credibility. It's a small contribution back to the project with a big impact.**
+
+Why add your company name?
+
+- Demonstrates the real-world usage of JSON Schema.
+- Provides visibility to organizations that have successfully adopted JSON Schema.
+- Encourages other organizations to adopt JSON Schema.
+- Builds trust in the JSON Schema ecosystem.
+
+This document also lists the organizations that use JSON Schema based on public information available in blog posts, events, and videos. If any organization would like to be added or get removed please make a Pull Request.
+
+Guidelines:
+- Please don't include your organization's logo or other trademarked material
+- Add a reference (link to a public blog post, video, slides, etc) mentioning that JSON Schema is used
+- Please include a contact link for maintainers
+
+Looking for extra credit? We also do [case studies](https://github.com/orgs/json-schema-org/projects/8/views/2) and accept sponsorship via [Open Collective](https://opencollective.com/json-schema).
+<details>
+<summary>Find out more about case studies</summary>
+If your company would like to share more about what you're doing in public, there's a good chance we'd love to collaborate on a case study.
+
+Please reach out to us via [our Slack](https://json-schema.org/slack) or creating a new Issue in this repo.
+
+From exerience, these case studies can take some time to develop, write, get approved, and published. It's best if you can find out for sure if you can publish a case study with us as early as possible. If you're at a big organization, you'll likely have to get approval from people who have no idea what you're talking about. Here's some context you can share with them.
+
+JSON Schema is an [OpenJS Foundation](https://openjsf.org/about) Project, under [The Linux Foundation](https://www.linuxfoundation.org/). Both the OpenJS Foundation and The Linux Foundation are registered non-profit organizations (501(c)(6)).
+
+By supporting JSON Schema with a case study, you are documenting its success and your smarts in picking it as a solution. You'll join the ranks of organizations like GitHub, Postman, Microsoft, and Zapier. JSON Schema case studies help justify the ongoing financial support required to mature, develop, and support the JSON Schema ecosystem. Case studies also demonstrate the strength of the JSON Schema ecosystem in production today. The next case study could be you.
+</details>
+
+| Organization | Contact | Description of Use / Reference |
+| --- | --- | --- |
+| GitHub | [Rachael Sewell](https://github.com/rachmari)<br/>[Robert Sese](https://github.com/rsese) | Github JSON Schema case study - [Validating documentation at GitHub](https://json-schema.org/blog/posts/github-case-study) |
```

Comment:
Postman belongs here as well to start I'm sure no? Or were you not trying to build the list up too much in this PR?

Reply:
Sounds good.